mailbox/api/src/main/java/org/apache/james/mailbox/model/Cid.java

```diff
@@ -47,6 +48,16 @@ private static String unwrap(String cidAsString) {
         return unwrapCid;
     }
 
+    public static Optional<Cid> fromRelaxedNoUnwrap(String cidAsString) {
```

Comment:
I'm trying to understand what this function does from its name and I must say I can't.
I would argue it requires a "policy" like, for example, guava Splitter